### DISCUSSION TITLE
【任务】【黑客松】WTF Solidity开源课程认证系统 ｜50,000 $PEOPLE

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#163] [任务 | Bounty]
目前WTF Solidity极简教程有43讲，也有很多人用这个教程学习。在路线图中，如果WTFSolidity repo超过1024个star，就会推出课程认证系统，给学习过的人发SoulBound Token。而目前已经有2000 star，是时候做认证系统了。

这个项目会参加THUBA（清华区块链协会）举办的黑客松（9月5日结束）。

**任务描述**：完成WTFSolidity课程认证系统的MVP（Minimal Viable Product）。包括但不限于以下内容：
1. 每一讲的随堂小测，参与[Solidity题库整理任务](https://github.com/AmazingAng/WTFSolidity/discussions/154)的贡献者默认加入这个任务 。
2. 认证系统产品设计
3. 认证系统UI设计。
4. 认证系统前端，主要是答题系统（对接谷歌表格quiz）。
5. 认证系统后端。
6. 对接SoulBound Protocol的SBT服务，给通过小测的人发放课程认证SBT。

**赏金**：50,000 $PEOPLE 代币 + 黑客松奖金，根据工作量分配。

请有意参加的WTF贡献者/成员回帖报名，并介绍下自己的技术栈（例如 前端：html/css/js, react）！另外如果还没填过WTF社区申请表的，也填一下：[表格](https://docs.google.com/forms/d/e/1FAIpQLSe4KGT8Sh6sJ7hedQRuIYirOoZK_85miz3dw7vA1-YjodgJ-A/viewform?pli=1)

**Comment by reborn-sama** [comment]
1

**Reply by AmazingAng** [reply]
介绍下自己的技术栈（例如 前端：html/css/js, react）

**Comment by jujube109** [comment]
1

**Reply by AmazingAng** [reply]
介绍下自己的技术栈（例如 前端：html/css/js, react）

**Comment by shenzhimeng** [comment]
有认证测试的吗

**Reply by AmazingAng** [reply]
什么认证测试？

**Comment by vecpeng** [comment]
1，前端

**Comment by Imatsea1997** [comment]
html/css/js/react

**Reply by AmazingAng** [reply]
你填下申请表吧 然后我拉你 https://wechat.wtf.academy/

**Reply by AmazingAng** [reply]
你wx多少？

**Comment by hyc6446** [comment]
前端:html/js/ts/vue
后端：python,golang

**Reply by AmazingAng** [reply]
你填下申请表吧 然后我拉你 https://wechat.wtf.academy/

**Reply by AmazingAng** [reply]
你wx多少？

**Reply by hyc6446** [reply]
Jason_Alex_He

**Comment by yumomei** [comment]
UI设计：
https://www.bilibili.com/video/BV11Q4y1o7ht?share_source=copy_web&vd_source=ca421f973c3d4d4fc562148a4c0513c4
https://www.bilibili.com/video/BV1x5411u7XD?share_source=copy_web&vd_source=ca421f973c3d4d4fc562148a4c0513c4
视频媒体宣传：
https://www.bilibili.com/video/BV1xY4y1A7mu?share_source=copy_web&vd_source=ca421f973c3d4d4fc562148a4c0513c4

**Reply by AmazingAng** [reply]
看起来很棒 你填下申请表吧 然后我拉你 https://wechat.wtf.academy

**Reply by yumomei** [reply]
你好，申请表网址进不去

**Reply by AmazingAng** [reply]
需要翻墙的 你微信多少？

**Reply by yumomei** [reply]
我在赛群里，昵称yumomei

**Comment by Coder-DjangoMo** [comment]
html/css/js/

**Reply by AmazingAng** [reply]
你wx多少？

**Reply by Coder-DjangoMo** [reply]
Django-Mo

**Comment by jujube109** [comment]
尴尬感觉测试仔没啥用，还是当学生算了😭



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2022年08月10日 14:06 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.******@***.***> |
| 主题 | Re: [AmazingAng/WTFSolidity] 【任务】【黑客松】WTF Solidity开源课程认证系统 ｜50,000 $PEOPLE (Discussion #163) |

介绍下自己的技术栈（例如 前端：html/css/js, react）

**Reply by AmazingAng** [reply]
测试肯定有用！

**Comment by stoneworld** [comment]
后端：Golang、Solidity

**Reply by AmazingAng** [reply]
你wx多少？

**Reply by stoneworld** [reply]
> 你wx多少？
w1113241948

**Comment by 0x3c** [comment]
front-end.
typescript/nodejs/reactjs

**Reply by AmazingAng** [reply]
Hey，你微信多少？
最好填一下报名表：wechat.wtf.academy

**Comment by Div-Denis** [comment]
html,css js nodejs react

**Reply by AmazingAng** [reply]
cool 你微信多少？

**Comment by GRITShannon** [comment]
前端 ： html css JavaScript 
后端： python  Java MySQL

**Reply by AmazingAng** [reply]
你好 需要先填表 https://docs.google.com/forms/d/e/1FAIpQLSe4KGT8Sh6sJ7hedQRuIYirOoZK_85miz3dw7vA1-YjodgJ-A/viewform

**Comment by pdshwc** [comment]
html/css/js/react/vue/nodejs

**Reply by AmazingAng** [reply]
你好 需要先填表 https://docs.google.com/forms/d/e/1FAIpQLSe4KGT8Sh6sJ7hedQRuIYirOoZK_85miz3dw7vA1-YjodgJ-A/viewform

**Comment by 0xFar777** [comment]
填了表单了，可以拉一下微信群吗兄弟

**Comment by zhengqian8958** [comment]
C, C++, Java, Python, Solidity
前端在学。。。不过html， css，js这些基本的内容都知道一些

**Comment by fchangjun** [comment]
前端：html/css/js/vue/react/小程序/工程化/混合开发/桌面端应用 后端：php,node

**Reply by AmazingAng** [reply]
没有填表

**Comment by chiguayeshao** [comment]
前端：React / TS / tailwindcss
后端：Node

**Comment by ADream-ki** [comment]
前端：react、ts

**Reply by ADream-ki** [reply]
还有图像视觉算法

**Comment by chuwj** [comment]
现在还能参加吗？
